mantidproject/mantid
# 37647 add a system test for polarizerefficiency algorithm

### Description of work

#### Summary of work
Add a system test for the PolarizerEfficiency algorithm.

Fixes #37647 

### To test:
- Check both new tests can be run by the `systemtest` script (you may need to explicitly re-run cmake and build the `SystemTestData` target).

*This does not require release notes* because **system tests are not user facing.**
